Fix of #16 Prettier error when [re]generating entity

smunir opened the pull request:
template updated for fix of #16. currentSearch property declaration added.

Commit:
Prettier error when [re]generating entity fixed

yelhouti commented:
@smunir it seems liks tests are not passing, if elastic search is not used the field should not be added, please add tests and try again. Thank you

smunir commented:
I just fixed compilation errors, couldn't manage to study your blueprint fully yet, I am afraid you would have to fix

Commit:
Minor bug fixes. Elasticsearch field removed. CSS classes added for fields.

Commit:
Merged with master from blueprint repository

Commit:
Merge branch 'master' of https://github.com/elhoutico/jhipster-primeng-blueprint

yelhouti commented:
@smunir could you rebase your PR please, we improved the design, I can't understand what your are trying to do

yelhouti commented:
@smunir please add some tests, you will see the problem with your code and will be able to fix it.

vivekmore commented on generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.html.ejs:
```diff
@@ -211,9 +211,10 @@ _%>
-                <button pButton type="button" id="cancel-save" icon="pi pi-ban" class="ui-button-secondary" (click)="previousState()" [label]="'entity.action.cancel' | translate"></button>
-                <button pButton type="submit" id="save-entity" icon="pi pi-save" [disabled]="editForm.invalid || isSaving" [label]="'entity.action.save' | translate"></button>
+                <br />
+                <button pButton type="button" id="cancel-save" icon="pi pi-ban" class="ui-button-secondary" (click)="previousState()" [label]="'entity.action.cancel' | translate" class="btn btn-secondary"></button>
```
The class attribute is repeated.
Consider putting all classes in one `class` attribute

smunir commented:
> @smunir could you rebase your PR please, we improved the design, I can't understand what your are trying to do

Yup I checked out the improvements. In fact I just fixed 2 compile errors in entity-management.component.ts.ejs
- Declared currentSearch: string;
- Added return in below snippet. Updated from 
```
if (this.currentSearch) {
    this.<%= entityInstance %>Service.search(this.<%= entityInstance %>Table.filters.global.value)
} else {
    return this.<%= entityInstance %>Service.query(lazyLoadEventToServerQueryParams(this.<%= entityInstance %>Table.createLazyLoadMetadata()));
}
```
to 

```
if (this.currentSearch) {
    return this.<%= entityInstance %>Service.search(this.<%= entityInstance %>Table.filters.global.value)
} else {
    return this.<%= entityInstance %>Service.query(lazyLoadEventToServerQueryParams(this.<%= entityInstance %>Table.createLazyLoadMetadata()));
}
```

You can fix them and discard my PR
